lighthouse-core/report/templates/report-template.html

```diff
@@ -23,14 +23,16 @@
   <meta name="viewport" content="width=device-width">
   <title>Lighthouse report: {{ url }}</title>
   <style>{{{ css }}}</style>
-  <script>{{{ script }}}</script>
+  {{{ script }}}
 </head>
 <body>
 
-<script>
-  // expose the original results
+<!--
```

Comment:
You're right. We'll need the JSON embedded for that use case. We can add it back later :)